accepted/00000-hub-reporting.md

```diff
@@ -26,30 +26,16 @@ let us stay with the good well known PostgreSQL database.
 Also PostgreSQL seems to be accepted in most Reporting tools as data source, while NoSQL databases
 are not supported, or only via standard DBMS modules. Native support was rarely available.
 
-## The Database
-We design for the possibility to use an external DB for the reporting, while in the default setup
-we consider to use the available PostgreSQL server on Uyuni. We will use a different database name
-and login data.
-The database needs to be made available on the network to either connect with the reporting tool
-or with the Hub to gather the data.
-The connection should be secured with SSL using the certificates we configure anyway for Uyuni Server.
+### The Database
+The main database is a PostgresDB in the hub system (but in the future it will be possible to 
+use also an external db): it stores all the information collected from all the server, and eventually
+aggregates them. Other databases with the same schema are also present in all the server, to collect 
+information for that system.
+The hub database needs to be made available on the network to either connect in a secure way (using the SSL
```

Comment:
I think not only the hub DB needs to be made accessible from the outside. I also every reporting DBs for the single server need that. At least if we want to syncronize the data using JDBC connections.

Reply:
right, tried to rephrase it